#!/usr/bin/python3
def raise_exception_msg(message=""):
    """raise_exception

        Return:
            Nothing
    """
    raise NameError(message)
